lib/src/widgets/address_text_field.dart

```diff
@@ -136,7 +135,7 @@ class AddressTextField extends StatelessWidget {
                               )),
                         )),
                   ],
-                  if (this.options.contains(AddressTextFieldOption.qrCode) && DeviceInfo.instance.isMobile) 
+                  if (this.options.contains(AddressTextFieldOption.qrCode) && ResponsiveLayoutUtil.instance.isMobile) 
```

Comment:
this needs to check by device type not just layout, since the QR scanner package is only available on iOS and Android